entries/b/bitbns.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Bitbns": {
+    "domain": "bitbns.com",
+    "tfa": [
+      "totp"
```

```suggestion
      "totp",
      "sms"
```
Based on the doc page, it looks like SMS 2FA is also available:
> The GAauth app creates a security code every minute, which you’ll use to login, instead of the SMS-based OTP.